src/App.js

```diff
@@ -18,43 +19,36 @@ class App extends React.Component {
   }
 
   componentDidMount = () => {
-      fetch("https://rancid-tomatillos.herokuapp.com/api/v2/movies")
-      .then(response => this.handleError(response))
-      .then(data => this.setState({movies: data.movies}))
-      .catch(error => console.log(error))
+    getAllMovies()
+      .then(data => this.setState({ movies: data.movies }))
+      .catch(error => this.setState({ error: error }))
   }
-  handleError = (response) => {
-    if (response.ok) {
-        return response.json()
-    } else if (!response.ok) {
-      // throw new Error (`Error`)
-      this.setState({error: response.status})
-    }
-  }
-  // Note: May use this code in the future to check for specific errors
-  // checkError(response) {
-  //   if (response.status === 422 || response.status === 500) {
-  //     // this.setState({error: response.status})
-  //     // this.decideErrorMsg()
-  //   } else {
-  //     return response.json()
-  //   }
+
+  // cleanData = (data) => {
```

Comment:
Curious if we could use two cleanData functions for both the Movie Container and Card